### PR TITLE
selenium: Remove webdriverio npm package

### PR DESCRIPTION
**Pull request by vaughnwalters**
selenium: Remove webdriverio npm package

The `webdriverio` package does not need to be an explicit dependency. It is a dependency of `@wdio/cli`.

Bug: https://phabricator.wikimedia.org/T325059

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,3800 +1,12 @@
 {
 	"name": "wdio-wikibase",
-	"version": "5.1.0",
-	"lockfileVersion": 2,
```
Comment (zeljkofilipin) [inline package-lock.json:4]: You probably changed the lockfileVersion by mistake.

**Review by zeljkofilipin — CHANGES_REQUESTED** [review]
Please revert the change to lockfileVersion.

File: package-lock.json (hunk 1 above)
Comment (vaughnwalters) [reply package-lock.json:4]: Yes you are correct, I have reverted this now @zeljkofilipin

**Comment by zeljkofilipin** [comment]
To make sure this didn't break anything, we have to run Wikibase, WikibaseLexeme and wikibase-termbox test suites with this change. According to [codesearch](https://codesearch-beta.wmcloud.org/search/?q=wdio-wikibase), those repositores use this package.

**Pull request closed without merging**